Microsoft.Toolkit.HighPerformance/Helpers/BitHelper.cs

```diff
@@ -103,6 +103,68 @@ public static bool HasLookupFlag(uint table, int x, int min = 0)
             return valid;
         }
 
+        /// <summary>
+        /// Checks whether the given value has any bytes that are set to 0.
+        /// That is, given a <see cref="uint"/> value, which has a total of 4 bytes,
+        /// it checks whether any of those have all the bits set to 0.
+        /// </summary>
+        /// <param name="value">The input value to check.</param>
+        /// <returns>Whether <paramref name="value"/> has any bytes set to 0.</returns>
+        /// <remarks>
+        /// This method contains no branches.
+        /// For more info, see <see href="https://graphics.stanford.edu/~seander/bithacks.html"/>.
```

Comment:
@michael-hawker do we need to add this to our license? It said to be in public domain, but I'm not sure.

Reply:
@Sergio0694 was there a specific snippet that was used there? We should probably link to the specific one. We should probably call this out in our third-party notice file if there was a direct usage of the code/algorithm.

Reply:
@Sergio0694 let's just update this line to be "For more background on this subject, see ..." and have the link point to the specific section with the anchor tag? I think we'll be fine then to merge this in after that.

Reply:
@michael-hawker Done in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3362/commits/4b6baa01936eca9c8461de3655fd52d0004f4a28 👍